catalogs/catalog-jdbc-common/src/main/java/org/apache/gravitino/catalog/jdbc/JdbcCatalogPropertiesMetadata.java

```diff
@@ -94,7 +95,11 @@ public class JdbcCatalogPropertiesMetadata extends BaseCatalogPropertiesMetadata
                 JdbcConfig.POOL_MAX_SIZE.getDefaultValue(),
                 true /* hidden */,
                 false /* reserved */));
-    PROPERTIES_METADATA = Maps.uniqueIndex(propertyEntries, PropertyEntry::getName);
+    PROPERTIES_METADATA =
+        ImmutableMap.<String, PropertyEntry<?>>builder()
+            .putAll(BASIC_CATALOG_PROPERTY_ENTRIES)
+            .putAll(Maps.uniqueIndex(propertyEntries, PropertyEntry::getName))
+            .build();
```

Comment:
@yuqi1129 can you please check this part?

Reply:
OK

Reply:
This part is fine with me.